src/firefly/js/metaConvert/MultiProductFileAnalyzer.js

```diff
@@ -324,7 +324,6 @@ function processAnalysisResult({table, row, request, activateParams, serverCache
     const imageParts= partAnalysis.filter( (pa) => pa.imageResult);
     let makeAllImageOption= !disableAllImageOption;
     if (makeAllImageOption) makeAllImageOption= imageParts.length>1 || (imageParts.length===1 && parts.length===1);
-    if (makeAllImageOption) makeAllImageOption= imageParts.every( (ip) => !ip.imageResult.override);
```

Comment:
I agree there is a bug here but I don't think this is the right fix. The line is meant to only enable the "all images" options if there are some parts that have not been replaced with an alternate image. The line above wrongly disables the all images option if _any_ of the parts have been replaced.

The "all images" option should only be disabled if _all_ the parts have been replace with an alternate image.
You will have to test it, but I think the following is a better fix than removing the line.

```js
if (makeAllImageOption) makeAllImageOption= !(imageParts.every( (ip) => ip.imageResult.override));
```

Reply:
> I agree there is a bug here but I don't think this is the right fix. The line is meant to only enable the "all images" options if there are some parts that have not been replaced with an alternate image. The line above wrongly disables the all images option if _any_ of the parts have been replaced.
> 
> The "all images" option should only be disabled if _all_ the parts have been replace with an alternate image.
> You will have to test it, but I think the following is a better fix than removing the line.
> 
> ```js
> if (makeAllImageOption) makeAllImageOption= !(imageParts.every( (ip) => ip.imageResult.override));
> ```

Yes, I understand that.  That was the question I was trying to ask in the meeting. @ejoliet said that as long as it is a multi-extension FITs, we just show all image option.  In this case, we no longer care if any part is replaced.

Reply:
This checks for case that we have not encountered. That is the case when we read a multi-extension FITS file and the replace very image with an alternate.

Reply:
In this case, your proposed change will work better.  Only all the parts are replaced, the makeAllImageOption=false.  I will use this change then.

Reply:
thanks.